BridgeAppSDK/SBAConsentDocumentFactory.swift

```diff
@@ -32,91 +32,12 @@
 //
 
 import ResearchKit
```

Comment:
No.  Only reason for pulling out ResearchUXFactory into it's own SDK is to untie from BridgeSDK (because for whatever reason it too confusing to use the SBB model objects and not the connection to the server.)  BridgeAppSDK is still intended to be dependent on ResearchKit and implements steps that are subclasses of `ORKStep`.